app/(admin)/admin/notifications/actions/sendEmailNotification.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+'use server';
+import { ID } from 'appwrite';
+import { messaging } from '@/api/serverConfig';
+
+/**
+ * Function to send email.
+ * @param props - subject, content.
+ * @param props.content - The email itself.
```

Comment:
The description here is a bit vague. Does this include the subject, who is receiving the email, and the email text/attachments?